src/Button.js

```diff
@@ -14,7 +16,8 @@ class Button extends React.Component {
 
     render() {
         var { accent, className, colored,
-            primary, raised, ripple, ...otherProps } = this.props;
+            primary, raised, ripple, component, href,
```

Comment:
Actually, no need to get the `href` since it will be inside `otherProps`. Since we do not make any changes to the value, that's fine.

But this could give an `eslint` error since it will be defined as a prop but "never" used? Could you test?